### DISCUSSION TITLE
Feature Request: Adopting the status text from the older version of the Add-on Manager offers a more accurate representation of the current state of the add-on, as well as its condition after an NVDA restart.

### DISCUSSION THREAD

**Discussion by cary-rowen** [nvaccess/nvda#15095] [Ideas]
### Is your feature request related to a problem? Please describe.
| Status text for Add-on Store | Status text for Add-on Manager |
| --- | --- |
| Installed, pending restart | Install, Enabled after restart |
| Pending removal | Enabled, Removed after restart |
| Disabled, pending restart | Enabled, Disabled after restart |

Firstly, I must clarify that I am not a native English speaker. Therefore, the terminology "Disabled, pending restart" is somewhat confusing for me as it leaves me unsure about whether the current status is "enabled". My apologies if this is a standard English phrase.
In contrast, the statement "Install, Enabled after restart" clearly informs me that the add-on is currently enabled and will switch to a disabled state following a restart.
### Describe the solution you'd like
If there are no additional issues to address, please adhere to the status text used in the previous version of the Add-on Manager. This includes any other status text entries not enumerated in the table above.
### Describe alternatives you've considered
Implement the current status text, with clarity provided through localized translations.
### Additional context
None

**Comment by seanbudd** [comment]
We prefer the current strings for the english interface. Converting this to a discussion to seek feedback from the rest of the community.
